src/index.d.ts

```diff
@@ -639,14 +681,21 @@ declare namespace admin.auth {
      */
     tenantId?: string | null;
 
+    /**
+     * The multi-factor related properties for the current user if available.
+     */
     multiFactor?: {
-      enrolledFactors: Array<{
-        uid: string;
-        phoneNumber: string;
-        displayName?: string;
-        enrollmentTime?: string;
-        factorId: string;
-      }>;
+
+      /**
+       * List of enrolled second factors on the current user record.
```

Comment:
List of second factors enrolled with the current user.

Reply:
Done.